bs-react-native-next/src/apis/NetInfo.rei

```diff
@@ -0,0 +1,62 @@
+module ConnectionType:
+  {
+    type t;
+
+    [@bs.inline "bluetooth"]
+    let bluetooth: t;
+
+    [@bs.inline "cellular"]
+    let cellular: t;
+
+    [@bs.inline "ethernet"]
+    let ethernet: t;
+
+    [@bs.inline "unknown"]
+    let unknown: t;
+
+    [@bs.inline "wifi"]
+    let wifi: t;
+
+    [@bs.inline "wimax"]
+    let wimax: t;
+  };
+module EffectiveConnectionType:
+  {
+    type t;
+
+    [@bs.inline "2g"]
+    let net2G: t;
+
+    [@bs.inline "3g"]
+    let net3G: t;
+
+    [@bs.inline "4g"]
+    let net4G: t;
+
+    [@bs.inline "unknown"]
+    let unknown: t;
+  };
+type info =
+    {. "effectiveType": EffectiveConnectionType.t, "_type": ConnectionType.t};
```

Comment:
Formatting looks weird in this file - did this go through refmt?

Could use a few newlines, too. :wink:

Reply:
Missed that due to the file type, fixed now.

Sorry about the assault on unsuspecting eyes!